build.gradle

```diff
@@ -22,7 +22,7 @@ buildscript {
     ]
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0-beta2'
+        classpath 'com.android.tools.build:gradle:3.0.0-rc1'
```

Comment:
that sneaked up with AS rc1 - should I remove it?

Reply:
I think this should be fine to keep in

Reply:
3.0.0 is stable now might as well switch to that if switching it.